community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/resume.py

```diff
@@ -78,23 +78,10 @@ def instance_properties(nodeset:object, model:str, placement_group:Optional[str]
         props.resourcePolicies = [placement_group]
 
     if reservation := lookup().nodeset_reservation(nodeset):
-        props.reservationAffinity = {
-            "consumeReservationType": "SPECIFIC_RESERVATION",
-            "key": f"compute.{util.universe_domain()}/reservation-name",
-            "values": [reservation.bulk_insert_name],
-        }
+        update_reservation_props(reservation, props)
 
-        if reservation.policies:
-            props.scheduling.onHostMaintenance = "TERMINATE"
-            props.resourcePolicies = reservation.policies
-            log.info(
-                f"reservation {reservation.bulk_insert_name} is being used with policies {props.resourcePolicies}"
-            )
-        else:
-            props.resourcePolicies = []
-            log.info(
-                f"reservation {reservation.bulk_insert_name} is being used without any policies"
-            )
+    if (fr := lookup().future_reservation(nodeset)) and fr.active and fr.specific:
+        update_reservation_props(fr.active_reservation, props)
```

Comment:
This change doesn't address my comment. 
We need to handle case of non-active FR before we decide to issue a request for this instance. 
Your current code just pretends that there is no reservation at all.

Reply:
Ah ok I initially misunderstood, yes hopefully this gap has now been addressed (in `resume_nodes` I purged all inactive nodes from the resume list). Behaviourally the node will go into configuring state for a few seconds before spitting out: 

```
srun: error: Job allocation 1 has been revoked
srun: Force Terminated JobId=1
srun: error: Node failure on whpcslurm-computenodeset-0